azure-pipelines/ui-automation/templates/flank/run-on-firebase-with-flank.yml

```diff
@@ -6,6 +6,9 @@ parameters:
   - name: testTargetPackages
     displayName: Packages as Test Targets
     type: string
+  - name: ignoredTarget
```

Comment:
Okay, looking at usage, it seems this can be used in any form by the caller and not really meant to denote "ignored" tests. I mean anyone can supply a value to this that says "package aaa.bbb.ccc" and that means those are now "included". So this is really down to the caller using this correctly and supplying something along the lines of either "notPackage or notAnnotation". Otherwise it just wouldn't do what we think it does. Is that right?

Reply:
Yes, that's right. I'm thinking of rolling this back because of that, and just hardcoding `notAnnotation org.junit.Ignore` within the `run-on-firebase-with-flank.yml` file, or include it at the end of the default value for `testTargetPackages` parameter